packages/react-app/components/TokenContract.tsx

```diff
@@ -0,0 +1,489 @@
+import { useInput } from ".";
+import { useContractKit } from "@celo-tools/use-contractkit";
+import { useEffect, useState } from "react";
+import { useSnackbar } from "notistack";
+import { truncateAddress } from "../utils";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import TextField from "@mui/material/TextField";
+import Divider from "@mui/material/Divider";
+import Grid from "@mui/material/Grid";
+import Typography from "@mui/material/Typography";
+import Link from "@mui/material/Link";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import IconButton from "@mui/material/IconButton";
+import SvgIcon from "@mui/material/SvgIcon";
+import { ethers } from "ethers";
+
+function HomeIcon(props) {
+  return (
+    <SvgIcon {...props}>
+      <path d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z" />
+    </SvgIcon>
+  );
+}
+
+const iconStyle = {
+  // backgroundColor: "#000000", // camel cased
```

Comment:
Commented out code like this should be deleted if it's not being used and doesn't provide any useful info.